ethereum/src/ERC20Manager.sol

```diff
@@ -248,54 +248,54 @@ contract ERC20Manager is
     }
 
     /**
-     * @dev Returns list of known VFT managers.
-     * @return knownVftManagers List of known VFT managers.
+     * @dev Returns list of VFT managers.
+     * @return vftManagers List of VFT managers.
      */
-    function knownVftManagers() external view returns (bytes32[] memory) {
-        return _knownVftManagers.values();
+    function vftManagers() external view returns (bytes32[] memory) {
+        return _vftManagers.values();
     }
 
     /**
-     * @dev Returns whether the VFT manager is known.
+     * @dev Returns whether the VFT manager is registered.
      * @param vftManager VFT manager address.
-     * @return isKnown `true` if the VFT manager is known, `false` otherwise.
+     * @return isVftManager `true` if the VFT manager is registered, `false` otherwise.
      */
-    function isKnownVftManager(bytes32 vftManager) external view returns (bool) {
-        return _knownVftManagers.contains(vftManager);
+    function isVftManager(bytes32 vftManager) external view returns (bool) {
+        return _vftManagers.contains(vftManager);
     }
 
     /**
-     * @dev Returns list of known tokens.
-     * @return knownTokens List of known tokens.
+     * @dev Returns list of tokens.
+     * @return tokens List of tokens.
      */
-    function knownTokens() external view returns (address[] memory) {
-        return _knownTokens;
+    function tokens() external view returns (address[] memory) {
+        return _tokens;
     }
 
     /**
-     * @dev Returns supply type of token.
+     * @dev Returns token type.
      * @param token Token address.
-     * @return supplyType Supply type of token. Returns `SupplyType.Unknown` if token is not registered.
+     * @return tokenType Token type. Returns `TokenType.Unknown` if token is not registered.
      */
-    function getTokenSupplyType(address token) external view returns (SupplyType) {
-        return _tokenSupplyType[token];
+    function getTokenType(address token) external view returns (TokenType) {
+        return _tokenType[token];
     }
 
     /**
-     * @dev Returns list of known bridging payments.
-     * @return knownBridgingPayments List of known bridging payments.
+     * @dev Returns list of bridging payments.
+     * @return bridgingPayments List of bridging payments.
      */
-    function knownBridgingPayments() external view returns (address[] memory) {
-        return _knownBridgingPayments.values();
+    function bridgingPayments() external view returns (address[] memory) {
```

Comment:
wouldn't it bug on huge amount of bridgingPayments? maybe accept amount and offset?

Reply:
should we also add pagination for tokens?